index.rst

```diff
@@ -333,42 +309,42 @@ The following level of integration tests (on the NCSA-integration test stand) ar
 .. TODO::
     DM has developed a way to do this, we should explore if this solution works for this case as well.
     For test data used in unit tests DM uses git-lfs to store repositories that are set up as eups packages.
-    Another possible solution is Travis, which is useds to test the LSST EFD helper class.
+    Another possible solution is Travis, which is used to test the LSST EFD helper class.
     Docker spins a temporary influxDB instance and loads test EFD data into it. A similar pattern could be loaded
     to test code that needs EFD data.
 
 
 .. _Tasks:
 
-Jobs for the Queue
-==================
+Scripts for the scriptQueue
+===========================
 
-The Queue is the mechanism to run Jobs in an automated fashion during commissioning and
-operations. The level of robustness required for these Jobs is divided among those still in development and those
+The scriptQueue is the mechanism to run scripts in an automated fashion during commissioning and
+operations. The level of robustness required for these scripts is divided among those still in development and those
 which are in full production.
 
 
-Jobs in Development
-^^^^^^^^^^^^^^^^^^^
-Jobs (scripts) undergoing development live in the `ts_queueJobsDevelop` repo. While in this repo, the Jobs are
+Scripts in Development
+^^^^^^^^^^^^^^^^^^^^^^
+Scripts undergoing development live in the `ts_scriptsDevelop` repo. While in this repo, the scripts are
```

Comment:
I think it would be good to add an example of why a script would ever be put in `ts_scriptsDevelop` rather then having it in a branch or something like that in the `ts_scripts` repo...